package.json

```diff
@@ -60,6 +62,12 @@
             "type": "string",
             "pattern": "^([0-9a-fA-F]{2}:){5}[0-9a-fA-F]{2}$"
           }
+        },
+        "checkPing": {
```

Comment:
Please set `checkPing` in the `config` object on line 50.

Reply:
This if fixed in the latest push, even though it's not marked as outdated (since I didn't change the schema).